### DISCUSSION TITLE
/stations/{stationId}/observations endpoint?

### DISCUSSION THREAD

**Discussion by TyceHerrman** [weather-gov/api#277] [Q&A]
Hello!

I've been exploring the api and trying to understand what one can and cannot do with it. I'm a little unclear on which endpoints are operational. It looks like on the specifications webpage that only the alerts/* endpoints are operational, but I see issues here on the github repo mentioning other endpoints, which makes me think others are working?

I'm trying to get historical observations for all stations in NYC. It sems like the /stations/{stationID}/observations endpoint is the one I should use. Experimenting with just one station ID, KNYC, on the specifications webpages with test start time of 2020-05-01T00:00:00-00:00 and end time of 2020-05-30T00:00:00-00:00 seems to just return an empty list. 

[response_1602004689961.zip](https://github.com/weather-gov/api/files/5335621/response_1602004689961.zip)

Happy to provide more information if needed.

**Comment by StephenClouse** [comment]
"Operational" in this context means it is a stable, 24/7 operationally-supported service. Currently the /alerts endpoints are operational. Other endpoints are functional but may be subject to temporary outages or data format changes with or without notice. We do, however, offer more limited support for this functionality.

The API does not retain historical data. Currently our retention policy across the board is a maximum of 7 days. Older observation data can be requested from [NCEI](https://www.ncei.noaa.gov/).
